### PR TITLE
change license to GPL and add liquidctl credits

### PR DESCRIPTION
**Pull request by arkste**

### CODE REVIEW

**Comment by arkste** [comment]
how about now? i've added copyright notices to the relevant files.

> If you do end up porting them ...

tbh, i've no plans in adding more devices or port your whole liquidctl package, i hope this should be fine by now.

more context why `coolctl` this exists: i was just playing around with [gousb](https://github.com/google/gousb) and ended up building this package basically for myself. 

i have a NZXT Kraken X62 in my linux machine and just wanted to throw in a binary to my autostart scripts without having to worry about dependencies or to install python.

**Comment by jonasmalacofilho** [comment]
> i have a NZXT Kraken X62 in my linux machine and just wanted to throw in a binary to my autostart scripts without having to worry about dependencies

It's always a bit weird to run Python code from sources in system services (e.g. the environment can matter, etc.).

For me personally things improved a lot once I started to use my distro's package manager: after I rebuild the package and install I know it will behave exactly like any other executable in the system.

What distro do you use, by the way?

File: README.md [hunk 1]
```diff
@@ -56,6 +56,17 @@ $ go run main.go speed pump 20 60  35 60  55 100  60 100
 $ go run main.go speed fan 20 25  35 25  50 55  60 100
 ```
 
-## Credits
+## License
 
-coolctl is a Golang-Port of [liquidctl](https://github.com/jonasmalacofilho/liquidctl) by [Jonas Malaco](https://github.com/jonasmalacofilho).
\ No newline at end of file
+coolctl – A cross-platform driver for NZXT Kraken X (X42, X52, X62 or X72).
```
Comment (jonasmalacofilho) [inline README.md]: Markdown: end with two spaces to enforce line break.

File: README.md [hunk 2]
```diff
@@ -56,6 +56,17 @@ $ go run main.go speed pump 20 60  35 60  55 100  60 100
...
+Copyright (c) 2019 Arkadius Stefanski
+
+coolctl is a Golang-Port of [liquidctl](https://github.com/jonasmalacofilho/liquidctl).
```
Comment (jonasmalacofilho) [inline README.md]: Same here and on the line bellow.

**Comment by arkste** [comment]
> It's absolutely fine by me.

can you review it one more time, please? 

sorry about causing that much trouble, shoulda've checked the license before, after reading a lot about GPL one could argue that `coolctl` is derived from `liquidctl`, therefore i don't want to cause any troubles and just want to get it right.

> What distro do you use, by the way?

i'm using Ubuntu 19.04 right now, but i do plan to switch the distro because of the changes in 19.10/20.04 (favoring snap packages over package manager packages).

what do you use as your daily driver?

**Comment by arkste** [comment]
thank you @jonasmalacofilho

**Comment by jonasmalacofilho** [comment]
> can you review it one more time, please?

Done! : )

> sorry about causing that much trouble, should've checked the license before, after reading a lot about GPL one could argue that coolctl is derived from liquidctl, therefore i don't want to cause any troubles and just want to get it right.

Thanks for getting it right though, and I'm really glad to see more people working on these types of things.

> i'm using Ubuntu 19.04 right now, but i do plan to switch the distro because of the changes in 19.10/20.04 (favoring snap packages over package manager packages).

I wasn't aware of that.  I mean, I've seen their promotion, but I wasn't aware they were being favored over regular packages.

> what do you use as your daily driver?

ArchLinux on the desktop and laptop, Ubuntu on servers.

**Comment by arkste** [comment]
> Thanks for getting it right though, and I'm really glad to see more people working on these types of things.

you're welcome :)

> I wasn't aware of that. I mean, I've seen their promotion, but I wasn't aware they were being favored over regular packages.

yep, see https://snapcraft.io/blog/chromium-in-ubuntu-deb-to-snap-transition for more informations.
while i do like the idea of the snap packages, i'm not a fan of the so called "transition"-process.

**Comment by jonasmalacofilho** [comment]
From the post it seems that chromium is a good case for this type of software distribution: it has a huge user base, security updates are critical, yet if it breaks it's not really that bad (the user can switch back to firefox for a few days).

Though I'm not sure if snaps are a good general solution to software distribution.  I find it  more reassuring to be able to _easily_ rebuild any package (and thus be able to apply patches to it or even debug it), which is why I like that in arch this is just a few keystrokes away.

**Pull request merged**